internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,157 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"testing"
+	"time"
+)
+
+func TestNewAsset(t *testing.T) {
+	type args struct {
+		asset *asset.Asset
+	}
+
+	newAsset, _ := asset.New().
+		NewID().
+		Team(id.NewTeamID()).
+		Name("test").
+		Size(10).
+		URL("test_url").
+		Build()
```

Comment:
```suggestion
	newAsset := asset.New().
		NewID().
		Team(id.NewTeamID()).
		Name("test").
		Size(10).
		URL("test_url").
		MustBuild()
```

MustBuild method can be used to panic if an error occurs. In unit testing, it is better to use these methods so that errors are not overlooked.